### ISSUE TITLE
Listing of organizations participating in CG

### ISSUE THREAD

**Issue by wseltzer** [w3c/cg-council#4]
I got a request for an enumeration of the organizations represented in a CG -- distinct from the sorting of the participants list by organization, rather: how many orgs are participating, and which?

**Comment by wseltzer** [comment]
in the meantime, here is a stupid bit of text manipulation on the generated html
`grep affiliation participants -C2|grep -v class |grep -v /p | sort | uniq -c | sort -gr`

**Comment by hober** [comment]
Here are a few related or perhaps duplicate requests:

1. Where the page currently says "N Non-Chair participants found", just say "N participants found." It's weird to have to do mental math to figure out how many participants there are.
1. Right below that, say "N organizations found."
1. Add a way to view the page which is organization-major, not person-major. A list of organizations, possibly with sub-lists of individual affiliated participants

**Comment by dontcallmedom** [comment]
With regard to @hober's requests, I think the "new" participants view pages shows all of what you're asking for, e.g. https://www.w3.org/groups/cg/privacycg/participants?sortaff=1

It doesn't give an easy access to the list of organizations without their associated participants (which I understand the original request was). @jean-gui is there any view in the new pages group that would give that listing?

Naturally, there is also the question of the duplication between the CG site view, and the regular W3C site view, e.g.
https://www.w3.org/community/privacycg/participants vs https://www.w3.org/groups/cg/privacycg/participants?sortaff=1

**Comment by jean-gui** [comment]
No, I don't believe there is such a view. It could likely be added fairly easily somewhere near https://www.w3.org/groups/cg/privacycg/participants?sortaff=1. Note that this page does show the number of organizations.
